test/e2e/data/ccm/ccm.yaml

```diff
@@ -655,6 +661,21 @@ rules:
   - apiGroups: [""]
     resources: ["persistentvolumeclaims/status"]
     verbs: ["patch"]
+  - apiGroups: [ "snapshot.storage.k8s.io" ]
+    resources: [ "volumesnapshotclasses" ]
```

Comment:
this change should go in the cluster-template-oci-addons file as well, and ccm.yaml

Reply:
updated